### PR TITLE
Adapter for phpredis/phpredis extension

### LINKED ISSUE #2: "predis/predis" package should be in "suggest" section not in "required"

### PR DESCRIPTION
**Pull request by korotovsky**
```
korotovsky@korotovsky:~/Development/redlock$ php vendor/bin/phpunit tests/
PHPUnit 4.6.4 by Sebastian Bergmann and contributors.

Configuration read from /home/korotovsky/Development/redlock/phpunit.xml.dist

........................................................

Time: 43.87 seconds, Memory: 7.75Mb

OK (56 tests, 323 assertions)
korotovsky@korotovsky:~/Development/redlock$ 
```

It also fixes https://github.com/everlution/redlock/issues/2

### CODE REVIEW

File: src/Adapter/PhpredisAdapter.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Everlution\Redlock\Adapter;
+
+use Everlution\Redlock\Adapter\AdapterInterface;
+use Everlution\Redlock\Exception\Adapter\InvalidTtlException;
+use RedisException;
+
+class PhpredisAdapter implements AdapterInterface
```
Comment (michcald) [inline src/Adapter/PhpredisAdapter.php]: I would have preferred it being named _PhpRedisAdapter_

File: tests/src/Adapter/PredisAdapterTest.php [hunk 2]
```diff
@@ -6,165 +6,9 @@
 /**
  * Integration test with Predis.
```
Comment (michcald) [inline tests/src/Adapter/PredisAdapterTest.php:7]: I am not sure why you have made updates to this file as it's the PredisAdapter

Comment (korotovsky) [inline tests/src/Adapter/PredisAdapterTest.php:7]: Adapter tests are now should extend `AdapterTest`: https://github.com/everlution/redlock/pull/3/files#diff-4197cb1219aa0eeca5adc3f83597824aR3

Basically most content of this test has been moved to this Abstract test

**Comment by korotovsky** [comment]
@michcald ping

**Comment by michcald** [comment]
Thanks for this, merging now.

**Comment by korotovsky** [comment]
Thanks!

**Pull request merged**